packit_service/worker/handler.py

```diff
@@ -140,9 +141,10 @@ class JobHandler:
     name: JobType
     triggers: List[JobTriggerType]
 
-    def __init__(self, config: Config, job: JobConfig):
+    def __init__(self, config: Config, job: JobConfig, event: Event):
         self.config: Config = config
         self.job: JobConfig = job
+        self.event = event
```

Comment:
+1, can't understand why I didn't do it then.

Reply:
no worries, tech debt is the responsibility of all of us :)